_authors/brianzheng.markdown

```diff
@@ -0,0 +1,8 @@
+---
+name: Brian Zheng
+short_name: brianzheng
+twitter: Brian48763045
+github: Frederic-Chopin
+---
+
+**<Name>** I'm a senior studying CS at UC Berkeley. I love to play soccer, go hiking, and play poker!
```

Comment:
Please insert your name on this line in the `<Name>` placeholder.

Reply:
Fixed ty!